ftva_lab_data/templates/view_items.html

```diff
@@ -0,0 +1,30 @@
+{% extends 'base.html' %}
+
+{% block content %}
+
+<h1>Items</h1>
+
+<!-- HTMX will listen for triggers from enclosed controls,
+then issue GET requests to 'render_table' URL
+and replace content of #table-container with result -->
+<form class="d-flex justify-content-end" hx-get="{% url 'render_table' %}" hx-target="#table-container"
+    hx-trigger="change from:find select, keyup changed delay:300ms from:find input">
+    <div class="d-flex w-50 gap-2">
+        <select name="column" class="form-select">
+            <option value="">All columns</option>
+            {% for field, label in columns %}
+            <option value="{{ field }}">{{ label }}</option>
+            {% endfor %}
+        </select>
+
+        <input id="search-input" class="form-control" type="text" name="search" placeholder="Enter search term...">
+    </div>
+</form>
+
+
+
+<!-- On page load, HTMX triggers GET request to 'render_table'
+and inserts it within div -->
+<div id="table-container" class="p-2" hx-get="{% url 'render_table' %}" hx-trigger="load"></div>
+
+{% endblock %}
```

Comment:
Please add a linefeed at the end of the last line, so the file ends with an empty line 31.